### ISSUE TITLE
Deploy fails with brand new meteor project (win and osx)

### ISSUE THREAD

**Issue by gerwinbrunner** [zodern/mup-aws-beanstalk#47]
I just created a new meteor project with `meteor create muptest`

then I followed the instructions in the mup-aws-beanstalk readme.

When I try to run the `mup setup` command, I get this error:

```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.setup [as handler] (/usr/local/lib/node_modules/mup/lib/plugins/docker/command-handlers.js:330:24)
    at PluginAPI._callee4$ (/usr/local/lib/node_modules/mup/lib/plugin-api.js:303:63)
    at tryCatch (/usr/local/lib/node_modules/mup/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/usr/local/lib/node_modules/mup/node_modules/regenerator-runtime/runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/local/lib/node_modules/mup/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/usr/local/lib/node_modules/mup/lib/plugin-api.js:73:191)
    at /usr/local/lib/node_modules/mup/lib/plugin-api.js:73:361
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:118:7)

```

The command `mup deploy` just does nothing and returns after a few seconds.

Any ideas what is going on here?

**Comment by gerwinbrunner** [comment]
the output of `mup deploy --verbose` is:

```
tmp/mup-test/.deploy# mup deploy --verbose
tmp/mup-test/.deploy#
```

**Comment by zodern** [comment]
Could you please show your config (`mup validate --scrub`) and mup version (`mup --version`)?

**Comment by gerwinbrunner** [comment]
mup version 1.4.5

I'm not sure what the issue was, but it only happened on a windows computer.
On a Mac I recreated the project and there it works now. So for me it is solved.

**Comment by one-pereyra** [comment]
I am having this issue. `mup validate --scrub` returns;
```
{
  "app": {
    "type": "aws-beanstalk",
    "name": "my-app",
    "path": "~/repos/pnyx17",
    "buildOptions": {
      "serverOnly": true,
      "debug": true
    },
    "auth": {
      "id": " - - - -- --  ",
      "secret": " - - - - - - -"
    },
    "minInstances": 1,
    "instanceType": "t2.micro",
    "sslDomains": [
      "sandpit.comm.care"
    ],
    "forceSSL": true,
    "region": "ap-southeast-2",
    "env": {
      "ROOT_URL": "https://subdomain.host.com",
      "MONGO_URL": "mongodb://user:pass@subdomain.host.com:27017/:27017 - - -- - - ",
      "MONGO_OPLOG_URL": "mongodb://user:pass@subdomain.host.com:27017/:27017 - - -- - - ",
      "MAIL_URL": "smtps://user:pass@smtp.mailgun.org:465/",
      "KADIRA_APP_ID": "------------",
      "KADIRA_APP_SECRET": "----------",
      "KADIRA_OPTIONS_PROXY": "https://-----------",
      "KADIRA_OPTIONS_ENDPOINT": "https://----------------",
      "PORT": 8081,
      "METEOR_SIGTERM_GRACE_PERIOD_SECONDS": 30
    },
    "maxInstances": 1,
    "oldVersions": 3
  },
  "plugins": [
    "mup-aws-beanstalk"
  ]
}
✓ Config is valid
```
`mup --version` is 1.4.5

**Comment by pagesrichie** [comment]
Client project due and I'm stuck at this step same as @one-pereyra .. what is going on?  Every time I try to set up it just runs into this issue as soon as it hits the "Configuring scaling" scaling part - every single time!: 

Finishing upload. This could take a couple minutes
=> Creating Version
=> Configuring Beanstalk Environment
 Created Environment
=> Waiting for Beanstalk Environment to finish launching
=> Configuring scaling
**InvalidParameterValue: Environment named mup-env-name is in an invalid state for this operation. Must be Ready.**

These are my specs:

I do not have a servers object.

mup validate --scrub
✓ Config is valid

npm -v
5.6.0

node -v
v8.11.1

So tested mup 1.4.5
With mup-aws-beanstalk 0.6.2
FAIL

Tested mup 1.4.5
With mup-aws-beanstalk 0.6.0
FAIL

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.0
FAIL

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.1
FAIL

At this point I have removed all the exports app variables except for the ones in the getting started in https://github.com/zodern/mup-aws-beanstalk/blob/master/docs/getting-started.md 

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.2
FAIL

Testing mup 1.4.5
With mup-aws-beanstalk 0.6.1
FAIL

Testing mup 1.4.5
With mup-aws-beanstalk 0.6.2
FAIL

Testing mup 1.4.0
With mup-aws-beanstalk 0.6.2
FAIL

Can't even deploy a new project!! If anyone has any advice on how to fix this or how they resolved it... would super appreciate this.

**Comment by pagesrichie** [comment]
Project due and I'm stuck at this step same as @one-pereyra .. what is going on?  Every time I try to set up it just runs into this issue as soon as it hits the "Configuring scaling" scaling part - every single time!: 

Finishing upload. This could take a couple minutes
=> Creating Version
=> Configuring Beanstalk Environment
 Created Environment
=> Waiting for Beanstalk Environment to finish launching
=> Configuring scaling
**InvalidParameterValue: Environment named mup-env-name is in an invalid state for this operation. Must be Ready.
    at Request.extractError (/usr/local/lib/node_modules/mup-aws-beanstalk/node_modules/aws-sdk/lib/protocol/query.js:47:29)
    at Request.callListeners (/usr/local/lib/node_modules/mup-aws-beanstalk/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
**
.. and so on...

These are my specs:

I do not have a servers object.

mup validate --scrub
✓ Config is valid

npm -v
5.6.0

node -v
v8.11.1

So tested mup 1.4.5
With mup-aws-beanstalk 0.6.2
FAIL

Tested mup 1.4.5
With mup-aws-beanstalk 0.6.0
FAIL

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.0
FAIL

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.1
FAIL

At this point I have removed all the exports app variables except for the ones in the getting started in https://github.com/zodern/mup-aws-beanstalk/blob/master/docs/getting-started.md 

Testing mup 1.4.4
With mup-aws-beanstalk 0.6.2
FAIL

Testing mup 1.4.5
With mup-aws-beanstalk 0.6.1
FAIL

Testing mup 1.4.5
With mup-aws-beanstalk 0.6.2
FAIL

Testing mup 1.4.0
With mup-aws-beanstalk 0.6.2
FAIL

Can't even deploy a new project!! If anyone has any advice on how to fix this or how they resolved it... would super appreciate this.
